fullscreen.bs

```diff
@@ -217,11 +225,26 @@ these steps:
   <p>As part of the next <a>animation frame task</a>, run these substeps:
 
   <ol>
-   <li><p>If either <var>error</var> is true or the <a>fullscreen element ready check</a> for
-   <var>pending</var> returns false, <a>fire an event</a> named <code>fullscreenerror</code> on
+
+   <li>
+    <p>If any of the following conditions are false, then set <var>error</var> to true:
```

Comment:
I don't think so, because conditions is plural. "If any following condition is false" would be grammatically correct, but less clear.